src/effects/chains/quickeffectchain.cpp

```diff
@@ -51,10 +51,23 @@ EffectChainPresetPointer QuickEffectChain::presetAtIndex(int index) const {
 }
 
 void QuickEffectChain::loadChainPreset(EffectChainPresetPointer pPreset) {
+    // Loading a chain preset sets the super knob's value to the value it was at
+    // when the chain preset was saved. It may be desirable to keep the knob's
+    // value as is, for instance when changing between multiple presets that
+    // contain a filter and an additional additional effect.
+    const double old_super_knob_value = getSuperParameter();
+
     EffectChain::loadChainPreset(pPreset);
-    if (pPreset) {
+
+    // The old value shouldn't be restored if this is the first time a preset is
+    // loaded or else the super knob will always be initialized to 0.0
```

Comment:
can this workaround be removed now that #11781 has been merged?

Reply:
Presumably. I'll try to find some time this weekend to check.

Reply:
It's indeed no longer necessary. Good catch.